_announcements/2016-05-05-mesh101.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "mesh.101 workshop"
+dateUntil: 2016-05-05
```

Comment:
I wonder if we could include more front-matter so the announcement can a bit more structured. I guess it'll just be the inclusion of location, map link, and time.